lib/rubycritic/source_control_systems/git.rb

```diff
@@ -52,6 +52,18 @@ def stashes_count
       def travel_to_original_state
         `git stash pop`
       end
+
+      def path_revisions_counts
+        `git log --name-only --format='' | sort | uniq -c`
```

Comment:
1. `uniq` is exclusive to unix machines so this won't run on windows.
2. The [option `follow`](https://www.git-scm.com/docs/git-log#git-log---follow) was used to provide a more accurate history of a file given that it tracks changes beyond renames. Without this option the `churn` score won't be as accurate as it used to be. Unfortunately, this option only works for a single file, so we can't use it here. This may be a trade-off we're willing to make or not...